### DISCUSSION TITLE
MA Server - Show version info in WEB UI

### DISCUSSION THREAD

**Discussion by Sonusss** [music-assistant/support#1330] [Feature requests and ideas]
Using docker images of the server I found hard to know which version is running...
Maybe I miss it but I can't find any version info from the Web UI
It would be nice to have it straight on the settings page for example

**Comment by Mr-HaleYa** [comment]
That's a very basic thing that should be exposed somewhere for all software.

**Reply by OzGav** [reply]
It’s a limitation of the docker setup. When using hassos it is shown on the addon page. We note the request but it is low priority at the moment.

**Reply by Mr-HaleYa** [reply]
Ya, I don't use docker, just realized for that it's already under settings. Solved for my needs 🙂 
![image](https://github.com/music-assistant/hass-music-assistant/assets/42821995/a9be7b01-878c-4a89-8dda-b580fdd89ea2)

**Comment by OzGav** [comment]
I think Marcel might have just added that!
